schema/src/main/java/org/hl7/v3/RCMRMT030101UK04Component02.java

```diff
@@ -180,6 +180,10 @@ public void setCompoundStatement(RCMRMT030101UK04CompoundStatement value) {
         this.compoundStatement = value;
     }
 
+    public boolean hasCompoundStatement() {
```

Comment:
Please add this class to schema's README in "Manually Modified classes"  section